packages/themes/src/reset.css

```diff
@@ -32,6 +32,7 @@
     outline-width: 1px;
     outline-style: none;
     outline-color: var(--scalar-color-accent);
+    outline-offset: -1px;
```

Comment:
I don't know if we want this everywhere 🤔

Reply:
yes actually it might feel too cluttered in the reference on second tests

Reply:
i'll just set it up in the sidebar items then

Reply:
proposing to set it only on button

Reply:
Sure